src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+/**
+ * Represents a command.
+ * Abstract parent class of all specific types of <code>Command</code>.
+ * A <code>Command</code> object corresponds to an action that can be executed.
+ */
+public abstract class Command {
```

Comment:
I like how you used Command as an abstract class for other child command classes.